pkg/controller/account/account_controller.go

```diff
@@ -316,7 +318,10 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		// see if in creating for longer then default wait time
 		if currentAcctInstance.IsCreating() && currentAcctInstance.IsOlderThan(createPendTime) {
 			errMsg := fmt.Sprintf("Creation pending for longer then %d minutes", utils.WaitTime)
-			r.setStateFailed(reqLogger, currentAcctInstance, errMsg)
+			stateErr := r.setStateFailed(reqLogger, currentAcctInstance, errMsg)
```

Comment:
(latent) This should return with no requeue.

Reply:
Updating the status will trigger a requeue anyway.

Reply:
Right, but
1. My main point was that this should return, not carry on. I think as it stands today we get lucky and the subsequent conditions prevent any further code from actually running.
2. We've talked about adding predicates to filter out status-update-only reconciles (cc @iamkirkbater). If we do that, it will be relevant whether our actual return statements request an explicit requeue. Of course, we would have to go curate all of them at that point anyway...